### PR TITLE
CI fix

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
molecule/default/tasks/taint.yml

### CODE REVIEW

**Comment by Akasurde** [comment]
cc @alinabuzachis @abikouo Could you please review this? This fixes CI failures.

**Review by abikouo — APPROVED** [review]

File: molecule/default/tasks/taint.yml [hunk 1]
```diff
@@ -315,6 +315,9 @@
       taints: "{{ taint_patch_1 }}"
     register: _result
 
+  - debug:
```
Comment (alinabuzachis) [inline molecule/default/tasks/taint.yml:318]: Do we want to keep this debug here? Other than that, LGTM!

**Review by alinabuzachis — APPROVED** [review]

File: molecule/default/tasks/taint.yml (hunk 1 above)
Comment (Akasurde) [reply molecule/default/tasks/taint.yml]: Thanks. removed.

**Comment by alinabuzachis** [comment]
recheck

**Comment by Akasurde** [comment]
recheck

**Pull request merged**